zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/tenant/TenantQueryControllerTest.java

```diff
@@ -743,9 +743,9 @@ public static Stream<Arguments> invalidTenantSearchQueries() {
                 """
                     {
                       "type": "about:blank",
-                      "title": "INVALID_ARGUMENT",
+                      "title": "Bad Request",
                       "status": 400,
-                      "detail": "Both after and before cannot be set at the same time.",
+                      "detail": "Failed to read request",
```

Comment:
❓ is this a generic error now generated from spring when trying to parse to `CursorForwardPagination`? What's the actual cause for failure is it violating the `@Size(min = 2, max = 300)` or that additional properties are present? Is that something we can actually keep by extending the validation in `SearchQueryPageRequestDeserializer`?